doc/man3/OSSL_HPKE_CTX_new.pod

```diff
@@ -84,48 +84,159 @@ is likely required before using these APIs.  HPKE is used by various
 other IETF specifications, including the (draft) TLS Encrypted Client 
 Hello (ECH) specification and others.
 
-HPKE is a highly flexible consstruct requiring supporting many optional
-inputs for the various combinations in which modes and algorithms can
-be used.
-
-Notes on the various operations supported:
-
-- OSSL_HPKE_seal() is used by senders to encrypt a value to a
-recipient (supplying optional additional authenticated data or
-AAD)
+HPKE is a highly flexible consstruct supporting many optional inputs for the
+various combinations in which modes and algorithms can be used.
+
```

Comment:
Maybe say something here about ... HPKE uses a combination of KEM, KDF and AEAD algorithms.

Reply:
Now says:
"HPKE is a standardised, highly flexible construct for encrypting "to" a public
key that supports combinations of a key encapsulation method (KEM), a key
derivation functions (KDF) and an authenticated encryption with additional data
(AEAD) algorithm, with optional sender authentication."